tests/client_tests.ts

```diff
@@ -47,6 +48,50 @@ describe.each([
     expect(client.config.host).toBe(customHost)
     expect(client.httpRequest.url).toBe(customHost)
   })
+  test(`${permission} key: No double slash when on host with domain and path`, async () => {
+    try {
+      const customHost = `${BAD_HOST}/api`
+      const client = new MeiliSearch({
+        host: customHost,
+        apiKey: key,
+      })
+      const health = await client.isHealthy()
+      expect(health).toBe(false) // Left here to trigger failed test if error is not thrown
+    } catch (e) {
+      expect(e.message).toMatch(`${BAD_HOST}/api/health`)
+      expect(e.type).toBe('MeiliSearchCommunicationError')
+    }
+  })
+
+  test(`${permission} key: host with double slash  should not double slash`, async () => {
+    try {
+      const customHost = `${BAD_HOST}//`
+      const client = new MeiliSearch({
+        host: customHost,
+        apiKey: key,
+      })
+      const health = await client.isHealthy()
+      expect(health).toBe(false) // Left here to trigger failed test if error is not thrown
+    } catch (e) {
+      expect(e.message).toMatch(`${BAD_HOST}/health`)
+      expect(e.type).toBe('MeiliSearchCommunicationError')
+    }
+  })
+
+  test(`${permission} key: host with one slash should not double slash`, async () => {
```

Comment:
What does it happen if we do not add any `/` at the end without no path (contrary to the first test)?
Maybe should we add a test for this?

Reply:
this is the default host that is used on every test
```javascript
const HOST = 'http://127.0.0.1:7700'
```
But maybe you prefer tests that clearly show we are testing this?

Reply:
No it's ok then!